src/vulkan-renderer/wrapper/device.cpp

```diff
@@ -0,0 +1,220 @@
+#include "inexor/vulkan-renderer/wrapper/device.hpp"
+
+namespace {
+// TODO: Make proper use of queue priorities in the future.
+constexpr float default_queue_priority = 1.0f;
+} // namespace
+
+namespace inexor::vulkan_renderer::wrapper {
+
+Device::Device(Device &&other) noexcept : device(other.device), graphics_card(other.graphics_card) {}
```

Comment:
We should be exchanging `device`.

Reply:
This won't work since `device` is needed in destructor.

Reply:
As you pointed out in chat, we can call `std::exchange` on the device. I will change it.

Reply:
Done.